client/frontend-tracks/index.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Internal dependencies
+ */
+import { recordUserEvent } from 'tracks';
+
+if ( window.wcPayFrontendTracks && window.wcPayFrontendTracks.length ) {
+	for ( const track of window.wcPayFrontendTracks ) {
+		recordUserEvent( track.event, track.properties );
+	}
```

Comment:
nit: Should we clear `wcPayFrontendTracks` after recording the event?

Reply:
Fixed on https://github.com/Automattic/woocommerce-payments/pull/9249/commits/218b1c81745f83f8b0cfe1cb9ac188e403709578.